### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Select Subscriptions_ Keyboard: Refresh button is not accessible using keyboard.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3341]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.1.1; #Keyboard; #Java tooling for IntelliJ/Eclipse; #C+E-H

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19
**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
**MAS Violated:** 2.1.1

**Repro Steps:**
1) Open IntelliJ IDEA application
2) Press 'Ctrl +Alt+ S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ in the Marketplace' and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Select Device login and tab till Sign in and press enter
8) Tab till 'Copy and Open' button and press enter
9) Paste the code in the browser and click on next.
10)Open the application again and navigate to all list items.
11)Verify keyboard focus is going to “refresh” button.

**Actual Result:**
Refresh button is not accessible using keyboard.

**Expected Result:**
Refresh button should be  accessible using keyboard.

User Impact: 
Keyboard users can not perform all operations if few controls are not accessible using keyboard.
MAS Reference: 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
![Bug_3341_Keyboard issue](https://user-images.githubusercontent.com/38523250/62206030-7c252000-b3ae-11e9-96e5-169dfc40a0cf.png)

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue that ActionButton & HideableDecorator in IntelliJ cannot get focus using tab, tracking here https://youtrack.jetbrains.com/issue/IDEA-191322. 

To solve this issue, there is a workaround that is reopen this dialog and subscription will be auto refreshed.

**Comment by Flanker32** [comment]
* **Importance of experience**: minimal. Once the dialog is open, it automatically refresh the subscriptions, so the feature is hardly ever used.
* **Quality of experience**: not impacted. A simple workaround: the user can re-open the dialog, it refreshes the list automatically. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Accessibility56 This is an external issue tracked in AAT, please help us verify it, thanks!

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749787
